### PR TITLE
Record validation refactor

### PR DESCRIPTION
**Pull request by minotogna**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "license": "MIT",
   "name": "open-foris-arena",
   "engines": {
-    "node": ">=10.13.0",
+    "node": "10.15.3",
```
Comment (fefrain) [inline package.json]: isn't better to use >=10.5.3 instead?

File: server/modules/record/manager/_recordManager/recordUniquenessValidator.js [hunk 2]
```diff
@@ -49,19 +47,19 @@ const validateEntityKeysUniqueness = (survey, record, nodeEntity) => {
   )(entityValidations)
 }
 
-const isDuplicateEntity = (survey, record, siblingEntitiesAndSelf, entity) => {
+const _isEntityDuplicated = (survey, record, siblingEntitiesAndSelf, entity) => {
```
Comment (fefrain) [inline server/modules/record/manager/_recordManager/recordUniquenessValidator.js]: it should be "_isEntityDuplicate" not "duplicateD"...

Comment (minotogna) [reply server/modules/record/manager/_recordManager/recordUniquenessValidator.js]: not sure: http://conjugator.reverso.net/conjugation-english-verb-duplicate.html

File: package.json (hunk 1 above)
Comment (minotogna) [reply package.json]: I'm not sure; but i have changed it. see next commit

File: server/modules/record/manager/_recordManager/recordUniquenessValidator.js (hunk 2 above)
Comment (fefrain) [reply server/modules/record/manager/_recordManager/recordUniquenessValidator.js]: duplicate is an [adjective](https://www.dictionary.com/browse/duplicated)!

Comment (minotogna) [reply server/modules/record/manager/_recordManager/recordUniquenessValidator.js]: changed!

**Pull request merged**